Libplanet.Net/Consensus/ConsensusReactor.cs

```diff
@@ -155,6 +156,80 @@ private void ProcessMessage(MessageContent content)
         {
             switch (content)
             {
+                case ConsensusVoteSetBitsMsg voteSetBits:
+                    // Note: ConsensusVoteSetBitsMsg will not be stored to context's message log.
+                    var messages = _consensusContext.HandleVoteSetBits(voteSetBits.VoteSetBits);
+                    try
+                    {
+                        var sender = _gossip.Peers.First(
+                            peer => peer.PublicKey.Equals(voteSetBits.ValidatorPublicKey));
+                        foreach (var msg in messages)
+                        {
+                            _gossip.PublishMessage(msg, new[] { sender });
+                        }
+                    }
+                    catch (InvalidOperationException)
+                    {
+                        _logger.Debug(
+                            "Cannot respond received ConsensusVoteSetBitsMsg message" +
+                            " {Message} since there is no corresponding peer in the table",
+                            voteSetBits);
+                    }
+
+                    break;
+
+                case ConsensusMaj23Msg maj23Msg:
+                    try
+                    {
+                        if (_consensusContext.HandleMessage(maj23Msg))
+                        {
+                            VoteSetBits voteSetBits = _consensusContext.Contexts[maj23Msg.Height]
+                                .GetVoteSetBits(
+                                    maj23Msg.Round,
+                                    maj23Msg.Maj23.BlockHash,
+                                    maj23Msg.Maj23.Flag);
+                            var sender = _gossip.Peers.First(
+                                peer => peer.PublicKey.Equals(maj23Msg.ValidatorPublicKey));
+
+                            _gossip.PublishMessage(
+                                new ConsensusVoteSetBitsMsg(voteSetBits),
+                                new[] { sender });
```

Comment:
If context already has sufficient messages, or on the higher step, we don't need to send `ConesneusVoteSetBitsMsg`, IMHO.